mtc-wasm/assets/js/js_ffi.js

```
@@ -0,0 +1,393 @@
+
+            async function onScanSuccess(decodedText, decodedResult) {
+                resolve(decodedText);
+                destroyHtml5QrcodeScanner();
+            }
+
+            activeHtml5QrcodeScanner.render(onScanSuccess);
+        } catch (error) {
+            console.error(error);
+            reject(error);
+        }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)